Update csc_assembly.md (#1429)
elixir-europe/rdmkit: pages/tool_assembly/csc_assembly.md

## For what can you use the CSC data management tool assembly?

> {% include image.html file="fi_csc_assembly.svg" caption="Figure 1. The CSC - IT Center for Science data management tool assembly." alt="CSC RDMkit" %}

Reviewer: if we do not use the old image anymore, I guess I can delete it!

{% include image.html file="fi_csc_assembly.svg" caption="Figure 1. The CSC - IT Center for Science data management tool assembly." alt="CSC RDMkit" %}

### Data management planning

> Research funders often require a data management plan (DMP) as part of the funding application process or after funding has been approved. See e.g. guidance on creating a [DMP for the Research Council of Finland (former Academy of Finland)](https://www.aka.fi/en/research-funding/apply-for-funding/how-to-apply-for-funding/az-index-of-application-guidelines2/data-management-plan/data-management-plan/). There are several tools to guide you through the planning process, such as [DMPTuuli](https://dmptuuli.fi/), a Finnish instance of DMPonline which includes guidance and templates provided by different organisations and funders. DMPs created in DMPTuuli are not yet machine actionable or linked to the CSC data management tool assembly services. However, a DMP is a valuable document when contacting and using the CSC research data management services. Other available DMP tools include [Data Stewardship Wizard](https://researchers.ds-wizard.org/wizard/), [ARGOS](https://argos.openaire.eu/splash/) and [DMPonline](https://dmponline.dcc.ac.uk/). Furthermore, research data support services at Finnish research organisations offer help and guidance through the planning process and when making decisions on data management.

Reviewer 2: ```suggestion
Research funders often require a data management plan (DMP) as part of the funding application process or after funding has been approved. See e.g. guidance on creating a [DMP for the Research Council of Finland (former Academy of Finland)](https://www.aka.fi/en/research-funding/apply-for-funding/how-to-apply-for-funding/az-index-of-application-guidelines2/data-management-plan/data-management-plan/). There are several tools to guide you through the planning process, such as [DMPTuuli](https://dmptuuli.fi/), a Finnish instance of DMPonline which includes guidance and templates provided by different organisations and funders. DMPs created in DMPTuuli are not yet machine-actionable or linked to the CSC data management tool assembly services. However, a DMP is a valuable document when contacting and using the CSC research data management services. Other available DMP tools include {%tool "data-stewardship-wizard" %} and {%tool "dmponline" %}. Furthermore, research data support services at Finnish research organisations offer help and guidance through the planning process and when making decisions on data management.
```

### Data processing and analysis 
For [processing](processing), [analysing](analysing) and [storing data](storage) during the research project, CSC offers several [computing platforms](https://research.csc.fi/computing). These include both environments for non-sensitive and [sensitive data](data_sensitivity). Depending on your needs, you can choose from a wide variety of computing resources: use [Chipster](https://chipster.csc.fi/) software for high-throughput data such as RNA-seq and single-cell RNA-seq, build your own custom virtual machine, or utilise the full power of our world-class supercomputers.

> CSC is hosting  EuroHPC's world-class supercomputer [LUMI](https://www.lumi-supercomputer.eu/), which is available for researchers across Europe for projects requiring extreme computing capacity. LUMI is one of the world's most competitive supercomputers and one of the most advanced platforms for artificial intelligence. In addition, you can utilize two national supercomputers Puhti and Mahti for medium to large scale simulations. Pouta and Rahti cloud computing services offer more flexibility, allowing the user to manage the infrastructure. CSC’s computers have a wide range of [preinstalled scientific software and databases](https://research.csc.fi/bioscience-programs) with usage instructions.

Reviewer 2: ```suggestion
CSC is hosting  EuroHPC's world-class supercomputer [LUMI](https://www.lumi-supercomputer.eu/), which is available for researchers across Europe for projects requiring extreme computing capacity. LUMI is one of the world's most competitive supercomputers and one of the most advanced platforms for artificial intelligence. In addition, you can utilise two national supercomputers Puhti and Mahti for medium to large-scale simulations. Pouta and Rahti cloud computing services offer more flexibility, allowing the user to manage the infrastructure. CSC’s computers have a wide range of [preinstalled scientific software and databases](https://research.csc.fi/bioscience-programs) with usage instructions.
```

> For sensitive data management, there is available [SD Connect](https://research.csc.fi/-/sd-connect) and [SD Desktop](https://research.csc.fi/-/sd-desktop) services. The Sensitive Data Services are designed to facilitate collaborative research across Finland and between Finnish academics and their collaborators.  SD Connect allows you to collect, organize and share your encrypted sensitive data in a secure manner via web browser or programmatically.  SD Desktop is a service that allows a user and their authorized colleagues to access a private computing environment workspace via a web browser and analyze the data within a secure cloud. A restricted version of SD Desktop is also available for processing health and social data for secondary use in compliance with the Finnish law and Findata regulation.

Reviewer 2: ```suggestion
For management of sensitive data, the [SD Connect](https://research.csc.fi/-/sd-connect) and [SD Desktop](https://research.csc.fi/-/sd-desktop) services are available. The Sensitive Data Services are designed to facilitate collaborative research across Finland and between Finnish academics and their collaborators.  SD Connect allows you to collect, organise and share your encrypted sensitive data in a secure manner via web browser or programmatically.  SD Desktop is a service that allows a user and their authorized colleagues to access a private computing environment workspace via a web browser and analyse the data within a secure cloud. A restricted version of SD Desktop is also available for processing health and social data for secondary use in compliance with the Finnish law and Findata regulation.
```

Reviewer 2: >in compliance with the Finnish law and Findata regulation.

Consider adding links for these.

Author: We have more links on the national page (Finland). But thank you for checking the text (we did try to check with 6 eyes already :D)

Reviewer 2: > We have more links on the national page (Finland). 

You could just say that more info is provided on the national page and link it directly. Up to you.

>But thank you for checking the text

No problem, it was a quick check. The more challenging bit is to move all the existing content to the new syntax for tools. I am trying to do it little by little as I get pages to review.

### Data sharing and publishing

> It is recommended to [publish](data_publication) data in data specific repositories. You can find many options from {% tool "elixir-deposition-databases-for-biomolecular-data" %}.  Furthermore, CSC and ELIXIR-FI will offer Federated EGA for sensitive human biomedical data that is linked to the central European Genome-phenome Archive.

Reviewer 2: ```suggestion
It is recommended to [publish](data_publication) data in data specific repositories. You can find many options from {% tool "elixir-deposition-databases-for-biomolecular-data" %}.  Furthermore, CSC and ELIXIR-FI will offer Federated EGA for sensitive human biomedical data that is linked to the central European Genome-phenome Archive.
```
```suggestion
It is recommended to [publish](data_publication) data in data specific repositories. You can find many options from {% tool "elixir-deposition-databases-for-biomolecular-data" %}.  Furthermore, CSC and ELIXIR-FI will offer Federated EGA for sensitive human biomedical data that is linked to the central {%tool "the-european-genome-phenome-archive" %}
```
